extensions/amp-subscriptions/0.1/local-subscription-platform.js

```diff
@@ -151,11 +151,18 @@ export class LocalSubscriptionPlatform {
   handleClick_(element) {
     if (element) {
       const action = element.getAttribute('subscriptions-action');
-      if (element.hasAttribute('subscriptions-service')) {
+      if (element.getAttribute('subscriptions-service') === 'local') {
+        this.executeAction(action);
+      } else if (!element.hasAttribute('subscriptions-service')
```

Comment:
Would it be just easier to do `if ((element.getAttribute('subscriptions-service') || 'auto') == 'auto')`?